tests/test_cli_compile.py

```diff
@@ -612,38 +630,31 @@ def test_local_url_package(
     assert dependency in out.stderr
 
 
-@pytest.mark.parametrize(
-    ("line", "dependency", "rewritten_line"),
-    (
-        pytest.param(
-            os.path.join(
-                MINIMAL_WHEELS_PATH, "small_fake_with_deps-0.1-py2.py3-none-any.whl"
-            ),
-            "\nfile:small_fake_with_deps-0.1-py2.py3-none-any.whl",
-            "file:small_fake_with_deps-0.1-py2.py3-none-any.whl",
-            id="Relative URL",
+def test_relative_file_uri_package(pip_conf, runner):
+    # Copy wheel into temp dir
+    shutil.copy(
+        os.path.join(
+            MINIMAL_WHEELS_PATH, "small_fake_with_deps-0.1-py2.py3-none-any.whl"
         ),
-        pytest.param(
-            os.path.join(
-                MINIMAL_WHEELS_PATH, "small_fake_with_deps-0.1-py2.py3-none-any.whl"
-            ),
-            "\nsmall-fake-with-deps"
-            " @ file://{absolute_path}/small_fake_with_deps-0.1-py2.py3-none-any.whl",
-            "small-fake-with-deps"
-            " @ file://{absolute_path}/small_fake_with_deps-0.1-py2.py3-none-any.whl",
```

Comment:
This one has been moved here: https://github.com/jazzband/pip-tools/pull/1453/files#diff-99b06d953a9978ee564df2e67547b8a76f3b1ab116ab7c89a9005b9239f2a904R590